packages/ndk/lib/domain_layer/usecases/relay_manager.dart

```diff
@@ -43,7 +43,7 @@ class RelayManager<T> {
   final EngineAdditionalDataFactory? engineAdditionalDataFactory;
 
   /// Are reconnects allowed when a connection drops?
-  bool _allowReconnectRelays = true;
+  bool allowReconnectRelays = true;
```

Comment:
why change to public?

Reply:
I had getter/setter which I use in Yana when going to background/foreground, but @franzaps is right that it can simple be public instead.